drizzle/0022_slippery_polaris.sql

```
@@ -0,0 +1,5 @@
+ALTER TABLE "documents" ADD COLUMN "hand_receipt_id" uuid;--> statement-breakpoint
+DELETE FROM "documents" WHERE "hand_receipt_id" IS NULL;--> statement-breakpoint
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)